src/main/protobuf/extvm/msg.proto

```diff
@@ -0,0 +1,142 @@
+syntax = "proto3";
+package io.iohk.ethereum.extvm;
+
+message CallContext {
+  bytes ownerAddr = 1;
+  bytes callerAddr = 2;
+  bytes originAddr = 3;
+  bytes contractCode = 4;
+  bytes inputData = 5;
+  bytes callValue = 6;
+  bytes gasPrice = 7;
+  bytes gasProvided = 8;
+  int32 callDepth = 9;
+  bytes receivingAddr = 10;
+  BlockHeader blockHeader = 11;
+  BlockchainConfig blockchainConfig = 12;
+}
+
+message BlockHeader {
+  bytes parentHash = 1;
+  bytes ommersHash = 2;
+  bytes beneficiary = 3;
+  bytes stateRoot = 4;
+  bytes transactionsRoot = 5;
+  bytes receiptsRoot = 6;
+  bytes logsBloom = 7;
+  bytes difficulty = 8;
+  bytes number = 9;
+  bytes gasLimit = 10;
+  bytes gasUsed = 11;
+  int64 unixTimestamp = 12;
+  bytes extraData = 13;
+  bytes mixHash = 14;
+  bytes nonce = 15;
+}
+
+message BlockchainConfig {
+  bytes frontierBlockNumber = 1;
```

Comment:
That depends on what IELE testnet is and whether there will be different configurable instances for it (like testnets for Ethereum mainnet). We don't need this at all, if we're only targeting a single network.

So, if we maintain ETC support, it might look like:

```
message BlockchainConfig {
  oneof config {
    EtcConfig etcConfig = 1;
    IELETestnet ieleTestnet = 2;
  }
}
```

where `IELETestnet` is empty.